ep/build-velox/src/build_velox.sh

```diff
@@ -10,7 +10,7 @@ BUILD_PROTOBUF=ON
 BUILD_TYPE=release
 VELOX_HOME=
 #for ep cache
-VELOX_REPO=https://github.com/oap-project/velox.git
+VELOX_REPO=https://github.com/zhztheplayer/velox.git
```

Comment:
Do you have PR for the Velox change?

Reply:
Not yes, I'll file one once this get approved.

Reply:
https://github.com/oap-project/velox/pull/154

I've open the PR. Would you want to review that one too?

Reply:
Got it. Thanks!

Reply:
Merged this PR.